portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,14 @@
+
+package com.google.sps.data;
+
+/** An object for a comment. */
+public final class Task {
```

Comment:
This class name is not relevant, so do update the class name in your later commit.